serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-openapi-services-with-wiremock.adoc

```diff
@@ -18,15 +18,15 @@ link:{wiremock_url}[WireMock] is an open source Mocking API for unit, integratio
 .Prerequisites
 The examples used in this document assumes that you have the following prerequisites:
 
-* Required tooling is installed. For more information, see link:{getting-familiar-with-our-tooling}[Getting familiar with our tooling].
-* A Serverless Workflow project is created and orchestrates OpenAPI services.
+* Required tooling is installed. For more information, see {getting-familiar-with-our-tooling}[Getting familiar with Serverless Workflow tooling].
```

Comment:
Here we need to use the product name I think

Reply:
Done